virttest/env_process.py

```diff
@@ -1135,6 +1133,7 @@ def preprocess(test, params, env):
         migrate_setup.migrate_pre_setup(dest_uri, params)
         # Map hostname and IP address of the hosts to avoid virsh
         # to error out of resolving
+        from virttest import virsh
```

Comment:
Seems that we could leverage the [lazy importing](https://docs.python.org/3.10/library/importlib.html?highlight=lazy_import#implementing-lazy-imports) feature to get rid of adding an explicit import statement in every place referred to that module. I just tried with the following example and it worked as expected.

So what do you think?

(p.s. looks that the [`_wrappers`](https://github.com/avocado-framework/avocado-vt/blob/master/virttest/_wrappers.py) could be the module the `lazy_import` function to be introduced to.)

```python
# /tmp/test/main.py 
import importlib.util
import sys


def lazy_import(name):
    spec = importlib.util.find_spec(name)
    loader = importlib.util.LazyLoader(spec.loader)
    spec.loader = loader
    module = importlib.util.module_from_spec(spec)
    sys.modules[name] = module
    loader.exec_module(module)
    return module


virsh = lazy_import("virsh")


if __name__ == "__main__":
    if len(sys.argv) > 1 and sys.argv[1] == 'libvirt':
        virsh.foo()
```
```python
# /tmp/test/virsh.py 
print("imported virsh")


def foo():
    print("calling virsh.foo")
```
```
$ python3 /tmp/test/main.py
(no output)

$ python3 /tmp/test/main.py libvirt
imported virsh
calling virsh.foo
```